deny.toml

```diff
@@ -0,0 +1,21 @@
+# https://embarkstudios.github.io/cargo-deny/cli/init.html
+
+[graph]
+all-features = true
+
+[licenses]
+allow = [
+    "MIT",
+    "Apache-2.0",
+]
+exceptions = [
+    { allow = ["Unicode-DFS-2016"], crate = "unicode-ident" },
+]
+
+[bans]
```

Comment:
I don't usually have bans and sources sections, are the defaults not good enough?

Reply:
It's spamming warnings about some duplicate versions of windows-sys.